### ISSUE TITLE
rtsp-simple-server executable not found

### ISSUE THREAD

**Issue by AkrithiDevadiga** [edgexfoundry-holding/device-usb-camera#360]
# 🐞 Bug Report

### Affected Services [**REQUIRED**]

 The issue is located in: Dockerfile (unable to pull an image of rtsp-simple-server)

## 🔥 Exception or Error
<pre><code>

level=ERROR ts=2024-10-25T06:17:53.443431456Z app=device-usb-camera source=init.go:86 msg="ProtocolDriver init failed: RtspServerExecutable file cannot be found: stat ./rtsp-simple-server: no such file or directory"
</code></pre>

![image](https://github.com/user-attachments/assets/45e9c8bf-701d-439b-97fc-3cbc4d54d350)
![image](https://github.com/user-attachments/assets/9a6b0cd3-8924-48f9-a055-c4ac411736c4)

## 🌍  Your Environment

**EdgeX Version [**REQUIRED**]:** Napa

**Anything else relevant?** It is showing in the dockerhub that, aler9/rtsp-simple-server this image is being replaced by https://hub.docker.com/r/bluenviron/mediamtx

![image](https://github.com/user-attachments/assets/526af73b-467f-4180-abc2-075fb6194fbc)

**Comment by cloudxxx8** [comment]
it should be fixed in the main branch
https://github.com/edgexfoundry/device-usb-camera/pull/334

**Comment by AkrithiDevadiga** [comment]
Yes, but it is being developed for v4 as I can see. What changes should I make to use it for version 3, i.e., napa?

**Comment by cloudxxx8** [comment]
you can open a PR and make the same change in the napa branch or in your local repository

**Comment by cloudxxx8** [comment]
close due to no activity
